data/json/mapgen/lake_buildings/islands.json

```diff
@@ -32,7 +32,7 @@
         "****~~**~~~//~~~~~~*****"
       ],
       "place_vehicles": [
-        { "vehicle": "boat_motor_single", "x": 4, "y": 5, "rotation": 180, "chance": 5, "status": 3 },
+        { "vehicle": "lake", "x": 4, "y": 5, "rotation": 180, "chance": 5, "status": 3 },
```

Comment:
same problem as the lighthouse, this boat can't fit into the map to spawn.

Reply:
I removed the 'lake' vehicle group spawn from there and restored boat_motor_single, i'll be adding a blank lake tile so the lake group can spawn there in the future.